app/scripts/modules/core/src/pipeline/config/pipelineConfigurer.html

```diff
@@ -35,7 +35,27 @@
     <div class="band band-info" ng-if="isTemplatedPipeline && !pipeline.locked">
       <span class="glyphicon glyphicon small glyphicon-lock"></span> Manual edits are not allowed on templated pipelines
     </div>
-
+    <div class="band band-info" ng-if="hasDynamicSource && pipelineExecutions && pipelineExecutions.length">
+      <span class="glyphicon glyphicon small glyphicon-wrench"></span> This template has a dynamic source. The
+      configuration is currently rendered using
+      <span ng-if="pipelineExecutions.length > 1" class="dropdown" uib-dropdown>
+        &nbsp;&nbsp;<button type="button" class="btn btn-xs btn-default dropdown-toggle" uib-dropdown-toggle>
+          {{ renderablePipeline.trigger.parentPipelineName || "Build #" + renderablePipeline.trigger.buildNumber }} <span class="caret"></span>
```

Comment:
I understand that it might be useful to use `parentPipelineName` or `buildNumber` to differentiate between executions, but there are executions that will not have either of these properties. I think the default has to be the timestamp of the execution.

Reply:
I'm using this quite a few places. And I guess it could be useful in other places as well. If I want to create a component, even if it end up just being used here, should I go the React route and create a React component and include it in angular using `react2angular`?

Reply:
Yes, that sounds good.

Reply:
Done 👍